src/SBarcodeGenerator.h

```diff
@@ -25,43 +28,116 @@ class SBarcodeGenerator : public QQuickItem
-        QML_ELEMENT
-#endif
+    #if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
+    QML_ELEMENT
+    #endif
 
 public:
+
+    /*!
+     * \fn explicit SBarcodeGenerator(QObject *parent)
+     * \brief Constructor.
+     * \param QObject *parent - a pointer to the parent object.
+     */
     explicit SBarcodeGenerator(QQuickItem *parent = nullptr);
-    ~SBarcodeGenerator() override {};
 
+    /*!
+     * \fn ~SBarcodeGenerator() override
+     * \brief Destructor.
+     */
+    ~SBarcodeGenerator() override { };
+
+    /*!
+     * \fn SCodes::SBarcodeFormat format() const
+     * \brief Returns current barcode format.
+     */
     SCodes::SBarcodeFormat format() const;
+
+    /*!
+     * \fn void setFormat(SCodes::SBarcodeFormat format)
+     * \brief Sets the barcode format.
+     * \param SCodes::SBarcodeFormat format - barcode format.
+     */
     void setFormat(SCodes::SBarcodeFormat format);
 
 public slots:
+
+    /*!
+     * \fn bool generate(const QString &inputString)
+     * \brief Function for generating barcodes from given string
+     * \param const QString &inputString - string of the barcode
+     */
     bool generate(const QString &inputString);
+
+    /*!
+     * \fn void setFormat(const QString &formatName)
+     * \brief Sets the barcode format.
+     * \param const QString &formatName - barcode format string.
+     */
     void setFormat(const QString &formatName);
+
+    /*!
+     * \fn bool saveImage()
+     * \brief Saves the generated barcode image.
+     */
     bool saveImage();
 
 signals:
+
+    /*!
+     * \brief This signal is emitted when barcode generation is finished. If any error happens, sends the error string to QML.
+     * \param const QString &error - error string.
+     */
     void generationFinished(const QString &error = "");
+
+    /*!
+     * \brief This signal is emitted to send width to QML.
+     * \param int width - width.
+     */
     void widthChanged(int width);
+
+    /*!
+     * \brief This signal is emitted to send height to QML.
+     * \param int height - height.
+     */
     void heightChanged(int height);
+
+    /*!
+     * \brief This signal is emitted to send margin to QML.
+     * \param int margin - margin.
+     */
     void marginChanged(int margin);
+
+    /*!
+     * \brief This signal is emitted to send eccLevel to QML.
+     * \param int eccLevel - error correction code level.
+     */
     void eccLevelChanged(int eccLevel);
+
+    /*!
+     * \brief This signal is emitted to send fileName to QML.
+     * \param const QString &fileName - fileName.
+     */
     void fileNameChanged(const QString &fileName);
 
+    /*!
+     * \brief This signal is emitted to send barcode format to QML.
+     * \param SCodes::SBarcodeFormat format - barcode format.
+     */
     void formatChanged(SCodes::SBarcodeFormat format);
 
 private:
-    int _width = 500;
-    int _height = 500;
-    int _margin = 10;
+    int _width    = 500;
+    int _height   = 500;
+    int _margin   = 10;
```

Comment:
C++ code rule 1 for member variables: `camelCase starting with 'm_', like std::string m_accessToken;.`